src/tests/DisplayCard.test.tsx

```diff
@@ -3,7 +3,7 @@ import { fireEvent, render, screen } from '@testing-library/react'
 import '@testing-library/jest-dom'
 import { DisplayCard } from '../components/AppMain/DisplayCard/DisplayCard'
 
-test('edit card', () => {
+test('try to edit and delete card', () => {
 	render(<DisplayCard front='front' back='back' _id={'123'} key={'123'} />)
 
```

Comment:
**issue:** this is incorrect test -  you should test your context provider if you want to test if deleting works correctly. Here you just render display card which is not complete integration of your implementation. 

PS. try to not mix 2 things in one test, I mean editing and deleting should be separate test cases